tests/HashTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace LanSuite\Tests;
+
+use LanSuite\PasswordHash;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function PHPUnit\Framework\assertEquals;
+
+class HashTest extends TestCase
+{
+
+    protected function setUp() : void
+    {
+        $GLOBALS['cfg'] = [];
```

Comment:
Lets try to avoid globals in unit tests at all.
If this is needed, we should work on a bit of a refactoring.